lib/PhasarLLVM/Plugins/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 file(GLOB_RECURSE PLUGINS_SRC *.h *.cpp)
-file(GLOB_RECURSE PLUGINS_SO *.cxx)
+#file(GLOB_RECURSE PLUGINS_SO *.cxx)
```

Comment:
Why have you out-commented this? Then the loop below is dead

Reply:
Because plugins are always build as shared which is an issue if you use conan to build it static linked. From an older discussion we dont use these plugins right now, so I uncommented it.